javascript-loops-practice/includes-seven.js

```diff
@@ -0,0 +1,14 @@
+/* exported includesSeven */
+
+function includesSeven(array) {
+  var index = [];
+  var i = 0;
+  while (i < array.length) {
+    index.push(i);
+    i++;
+  }
+  if (i === 7) {
+    return true;
+  }
```

Comment:
You can get rid of your array and move this conditional statement into your `while` loop.